conanfile.py

```diff
@@ -94,7 +94,7 @@ def requirements(self):
                 "libunwindstack/80a734f14@{}#0".format(self._orbit_channel))
         self.requires("zlib/1.2.11#9e0c292b60ce77402bd9be60dd68266f")
 
-        if self.options.with_gui and self.options.with_crash_handling:
+        if self.options.with_orbitgl:
```

Comment:
Since crashpad is not optional anymore, can you remove the `with_crash_handling` option?